src/main/scala/scynamo/Scynamo.scala

```diff
@@ -16,4 +16,8 @@ trait ScynamoFunctions {
 
   def decodeQueryResponse[A: ObjectScynamoDecoder](response: QueryResponse): EitherNec[ScynamoDecodeError, List[A]] =
     response.items().asScala.toList.traverse(ObjectScynamoDecoder[A].decodeMap(_))
+
+  def decodeScanResponse[A: ObjectScynamoDecoder](response: ScanResponse): EitherNec[ScynamoDecodeError, List[A]] =
+    response.items().asScala.toList.traverse(ObjectScynamoDecoder[A].decodeMap(_))
```

Comment:
Hmm I just had a thought - should we use
```suggestion
    response.items().asScala.toList.parTraverse(ObjectScynamoDecoder[A].decodeMap(_))
```

The difference - reporting errors for all items vs short circuit on the first problem 🤔

Reply:
Also applies to `decodeQueryResponse` of course

Reply:
you mean that would still try  to decode all items and `maybe` return a list full of error `or` a list of mixed errors and decoded items, instead of stoping with the first error it finds? 🤔 

Not against, also don't a great the advantage. In any case... we should then change the implementation on `QueryRepsonse` I guess, to follow a similar pattern?

Reply:
There is no mixing of errors possible but let's say there are two items that would fail to be decoded:
 * `traverse` would give you only the errors from the first item but would not try to decode the second one
 * `parTraverse` would give you the errors from both items but it would have to try and decode all of them

Reply:
There is no right and wrong here, just not sure what we prefer and since I don't have a use case up to you

Reply:
that was a good explanation! 👍🏼 
🤔  I still would opt for just `traverse` (?) As soon as you find an error, return it and save trying to decode the rest. Because the other way, you are also going to return errors, only that maybe more of one. 

I would see an advantage if there were a mixing of errors and items possible, because then the user can choose to care about the decoded stuff and just dump the errors, for example. But if that is not possible.. dunno,  I would still go for 1 (?)

Reply:
That is fine by me 👍